### ISSUE TITLE
(Not strictly an issue) How to specify a placeholder in a filename

### ISSUE THREAD

**Issue by macalimlim** [cargo-generate/cargo-generate#255]
How to specify a placeholder in a filename? for example I want a file to be named same as the `crate_name` placeholder. I see in an earlier version this was supported, and it was removed in a later version. Why was this removed?

**Comment by k0pernicus** [comment]
`{{crate_name}}` is still commonly used in our template and works with the current version 🤔 

I think I did not understood fully your issue...

**Comment by macalimlim** [comment]
Example, I want to have a file named like `{(crate_name}}.txt` within my project folder...

**Comment by sassman** [comment]
I think that is very common case, that placeholders are used as part of a file or even directory name.

**Comment by k0pernicus** [comment]
Sorry @macalimlim, I read too fast the issue description...

Indeed, we added this via this PR: https://github.com/ashleygwilliams/cargo-generate/pull/161, that I wrote.  
However, there were some issues with this PR: https://github.com/ashleygwilliams/cargo-generate/pull/174, that you can read here: https://github.com/ashleygwilliams/cargo-generate/pull/182.

The reasons are still valid (imoo), and some work need to be done before going back to #161.
You can propose a solution here @macalimlim if you want to work on it :)

**Comment by k0pernicus** [comment]
Closing it as duplicate of #159

**Comment by MarcoIeni** [comment]
> Closing it as duplicate of #159

The issue is still opened. Should this be closed?
